src/components/component-playground.test.js

```diff
@@ -3,6 +3,14 @@ import { render, shallow } from 'enzyme';
 import ComponentPlayground, { PlaygroundProvider } from './component-playground';
 
 describe('<ComponentPlayground />', () => {
+  beforeEach(() => {
+    window.localStorage = { setItem: () => {} };
```

Comment:
Let's not permanently destroy `localstorage` for the other tests.

suggest at the top:

```
const realStorage = localStorage;

// ...

afterEach((() => {
  window.localStorage = realStorage;
});
```

Reply:
Actually, if `setItem()` is all you need to override, just `sandbox.stub()` it.

Reply:
@ryan-roemer I just copied this from the manager's tests :wink: I think we're testing in an environment where the local storage is not mocked out-of-the-box anyway

Reply:
I'm not totally following. The `window.localStorage = undefined;` is a potential footgun down the road for unrelated tests that may use `localStorage` for something innocuous...

Reply:
If we're super sure of that, then as a protective measure, I suggest a global `beforeEach` that sets `window.localStorage = undefined;` so we discover any kinks immediately in the future...

Reply:
Depends; we'd have to update it here as well: https://github.com/FormidableLabs/spectacle/blob/master/src/components/manager.test.js#L65

Reply:
Let's do that. It's a footgun here and in the existing code, so we'll just take care of both.

Reply:
@ryan-roemer Is this what you had in mind? https://github.com/FormidableLabs/spectacle/pull/407/commits/af1c727b0af1b4a0055f2b5d5407bdad82f8214c

Reply:
Yep -- or maybe move `const origLocalStorage = window.localStorage;` above the first `describe`, so that you're sure no tests have mucked with things. And you only have to set it once. But your call there -- as you have it already is fine too...

Reply:
Good point!